tests/onmount.test.tsx

```diff
@@ -1,8 +1,19 @@
-import React, { Fragment, Suspense } from 'react'
+import React, { Suspense } from 'react'
 import { act, fireEvent, render, waitFor } from '@testing-library/react'
 import { Provider as ProviderOrig, atom, useAtom } from '../src/index'
 
-const Provider = process.env.PROVIDER_LESS_MODE ? Fragment : ProviderOrig
+const Provider = process.env.PROVIDER_LESS_MODE
+  ? (props: any) => props.children
+  : ProviderOrig
+
+let savedNodeEnv: string | undefined
+beforeEach(() => {
+  savedNodeEnv = process.env.NODE_ENV
+  process.env.NODE_ENV = 'production'
+})
+afterEach(() => {
+  process.env.NODE_ENV = savedNodeEnv
+})
```

Comment:
So, does this mean onMount has some issues with DEV mode? Or, is it just a testing thing?
Needs investigation. Help wanted.